encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -0,0 +1,168 @@
+function runTest(config, testname) {
+
+    var testname = config.keysystem + ', successful playback, persistent-license, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', set src before setMediaKeys, check events';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
```

Comment:
Is there a reason for the three spaces after the '{' here, before the '?' on line 33, etc.? This just wastes whitespace, especially for longer lines on smaller displays.